docs/docsite/rst/dev_guide/developing_modules_checklist.rst

```diff
@@ -24,7 +24,8 @@ To contribute a module to Ansible, you must:
 * minimize module dependencies
 * support :ref:`check_mode <check_mode_dry>` if possible
 * ensure your code is readable
-* if a module is named `<something>_facts`, it is because it is returning `ansible_facts`. Do not name modules that do not do this with `_facts`.
+* if a module is named ``<something>_facts``, it should be because it's main purpose is returning ``ansible_facts``. Do not name modules that do not do this with ``_facts``.
```

Comment:
```suggestion
* if a module is named ``<something>_facts``, it should be because its main purpose is returning ``ansible_facts``. Do not name modules that do not do this with ``_facts``.
```